test/tall/invocation/chain_comment.stmt

```diff
@@ -96,4 +96,5 @@ target // c
 .prop = value;
 <<<
 target // c
-    .prop = value;
```

Comment:
Nit: This looks kinda weird. Any plans to fix this? Or is it just another comment thing that probably won't be hit as much.

Reply:
It's a little wonky, but I don't have any concrete ideas on how to improve it. For reasons I don't totally understand, the old formatter seemed to do a generally better job with indentation when comments appeared in unexpected locations. The new formatter, maybe because it tries harder to cram things into a nice AST-ish Piece tree structure, seems to get tripped up more easily by comments in weird spots.

Fortunately, users rarely put comments in weird places, so it mostly works out. There are sometimes bugs filed, though, and fixing them feels a bit like wack-a-mole.